bin/verify-version-consistency.php

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env php
+<?php
```

Comment:
This is really useful. It might be nice to add something like this in `wp-dev-lib`.